integration/integration.go

```diff
@@ -28,8 +32,8 @@ func GetIntegrationName(result string) string {
 	id := strings.Split(resultSplit[1], "]")[0]
 
 	res, err := LwClient.Integrations.Get(id)
-	if err != nil {
-		log.Fatalf("Unable to find integration id: %v", id)
+	if err != nil || len(res.Data) == 0 {
```

Comment:
I found a panic when we don't fine any integration and we try to access `.Data[0].Name`